Terminal.Gui/Views/ColorPicker.cs

```diff
@@ -235,18 +247,22 @@ private void DisposeOldViews ()
         }
     }
 
-    private void RebuildColorFromBar (object? sender, EventArgs<int> e) { SetSelectedColor (_strategy.GetColorFromBars (_bars, Style.ColorModel), false); }
+    /// <exception cref="Exception">A delegate callback throws an exception.</exception>
+    private void RebuildColorFromBar (object? sender, EventArgs<int> e)
+    {
+        SetSelectedColor (_strategy.GetColorFromBars (_bars, Style.ColorModel), false);
+    }
 
+    /// <exception cref="Exception">A delegate callback throws an exception.</exception>
     private void SetSelectedColor (Color value, bool syncBars)
     {
         if (_selectedColor != value)
         {
-            Color old = _selectedColor;
+            RaisePropertyChanging (this, new (nameof (SelectedColor)));
```

Comment:
See above.

Reply:
Resolving this one in favor of the other one to keep it in one place.